tests/post/steps/test_service_configuration.py

```diff
@@ -219,7 +243,17 @@ def get(self, name, namespace):
                 "Unable to read {} ConfigMap  in namespace {} with error: {!s}"
                 .format(name, namespace, exc)
             )
-        return response
+
+        try:
+            csc_key = "config.yaml"
+            csc_object = yaml.safe_load(response.data[csc_key])
+        except yaml.YAMLError as exc:
+            raise Exception(
```

Comment:
As a general practice, only put the expression that is expected to break within a `try` block. This makes it easier for the reader to understand what part of your code is supposed to raise the exception that you're catching with `except`.

In this case, it'd be nice to add another guard against the absence of the `config.yaml` key:

```suggestion
        csc_key = "config.yaml"
        csc_data = response.data.get(csc_key)
        if csc_data is None:
            raise Exception("Missing {} key in ConfigMap".format(csc_key))

        try:
            csc_object = yaml.safe_load(csc_data)
        except yaml.YAMLError as exc:
            raise Exception(
```

Reply:
Yeah... I know, just copy/pasted what was below, I should have changed it...